web-client/src/global.scss

```diff
@@ -27,8 +27,8 @@
 
 :root {
   // Docs: https://ionicframework.com/docs/theming/themes#application-colors
-  --ion-background-color: var(--ion-color-prussian-blue);
-  --ion-background-color-rgb: var(--ion-color-prussian-blue-rgb);
+  --ion-background-color: var(--ion-color-primary);
+  --ion-background-color-rgb: var(--ion-color-primary-rgb);
```

Comment:
We should rename this to avoid a conflict between the background and primary colors in Ionic's color system: Ionic expects `primary` to be usable on top of the background color. (If they're the same, a lot of Ionic's assumptions break down, and will render wrong.)

For the Nautilus theme, the Ionic `primary` should probably be defined to be Portland Orange, and the background (Prussian Blue) should be named something other than `primary`, such as `primary-background`.

This diff implements that suggestions: cf254e2 – how does that look?

Reply:
I'm more comfortable with implementing this on a different PR.

Reply:
#33

Reply:
I think it makes more sense to fix this in this PR?

I added a story and pushed some temporary branches to demonstrate the issue more clearly:

1. [Baseline](https://www.chromatic.com/test?appId=60ddb6c36599fd003b211fd8&id=60fe7994ca4d240039ee58de) - How the theme looks like on main (with the stepped color fix from this branch).
2. [Without fix](https://www.chromatic.com/test?appId=60ddb6c36599fd003b211fd8&id=60fe7afcca4d240039ee94bd) - How the branch currently looks: in particular, see how the primary / background color conflict doesn't just affect the default button styling, but makes outline buttons and links completely invisible. (It probably breaks more than this: every Ionic component that references the primary color will render unexpectedly.)
3. [With fix](https://www.chromatic.com/test?appId=60ddb6c36599fd003b211fd8&id=60fe7b00db358200391164c9) - How this branch looks with the suggested fix applied.

We don't necessarily have to use that particular fix, as long as we do something that doesn't break the Ionic color system's assumptions?